CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.9.0] - 2020-08-26
```

Comment:
I'm all for this change, as it `will be deprecated`.
But then I will consider this as a breaking change in the SDK, as customers might need to change their code if they'll upgrade to this version.
So 2.0.0 would be better suited here.

Reply:
But this also mean that the definition of `LocalizableParam` could be removed.

Reply:
Agree, will update